src/cephadm/tox.ini

```diff
@@ -70,7 +70,7 @@ deps =
   {[testenv]deps}
   pytest-cov
 commands =
-  pytest -v --cov=cephadm --cov-report=term-missing --cov-report=html {posargs}
+  pytest -v --cov=cephadm --cov=cephadmlib --cov-report=term-missing --cov-report=html {posargs}
```

Comment:
Does this mean we could now have a `src/cephadm/cephadmlib/tests` dir with test files in it and it would run them?

Reply:
What this option does is to add cephadmlib and the files under it to the coverage report. it doesn't change the test selection.